modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -218,18 +223,47 @@ export default class AdvancedOpts {
             .classed( 'group-toggle-caret-wrap', true)
             .append( 'div' )
             .attr( 'class', 'adv-opt-toggle' )
-            .classed( 'combobox-caret', d => d.members.length )
-            .on( 'click', function(d) {
-                if (d.members.length) {
-                    let body      = d3.select( `#${ d.name }_group` ).select( '.group-body' ),
-                        bodyState = body.classed( 'hidden' );
-
-                    body.classed( 'hidden', !bodyState );
-                    body.classed( 'keyline-bottom', bodyState );
-                }
-            });
+            .classed( 'combobox-caret', d => d.members.length );
 
         caretWrap.merge(caretWrapEnter);
+        caretWrap.on( 'click', instance.showBody);
+    }
+
+    showBody(d) {
+
+        if (d3.event.target.classList.contains( 'conflate-type-toggle' )) return;
+
+
+        if (d3.event.target.classList.contains( 'adv-opts-group-title' )) {
+            let input = d3.select( `#${d.name}-toggle` ),
+                shouldHide = !input.empty() && !input.property('checked');
+            instance.toggleOption(d, shouldHide, true);
+        } else if (d.members.length) {
+            let bodyState = d3.select( `#${ d.name }_group .group-body` ).classed( 'hidden' );
+            d3.selectAll('.advanced-opts-content .form-group .group-body')
+                .classed('hidden', function(data) {
+                    if (data.name === d.name) {
+                        let disabled = d3.select(this.parentElement)
```

Comment:
I am not sure if others have feelings on this, but this makes the behavior that if you have an accordion open, clicking that accordion's checkbox or label acts as a noop. you can only close it and then disable it (why you might click the checkbox) if you either click on its caret/whitespace or open up another accordion.